client/src/core/services/parse-map.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ParseMapService } from './parse-map.service';
+
+describe('ParseMapService', () => {
+  let service: ParseMapService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
```

Comment:
Codacy found an issue: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5943029)